packages/mongo/mongo_driver.js

```diff
@@ -1106,6 +1106,26 @@ _.extend(SynchronousCursor.prototype, {
   }
 });
 
+SynchronousCursor.prototype[Symbol.iterator] = function () {
+  var self = this;
+
+  // Get back to the beginning.
+  self._rewind();
+
+  return {
+    next: function () {
+      var doc = self._nextObject();
+
+      if (!doc) {
+        return {done: true};
+      }
+      else {
+        return {value: doc};
```

Comment:
Let's add `done:false` explicitly here.

Reply:
We do not have it in [Minimongo version](https://github.com/meteor/meteor/blob/devel/packages/minimongo/cursor.js#L107) and it is [not required by the standard](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterator_protocol):

> Has the value false if the iterator was able to produce the next value in the sequence. This is equivalent of not specifying the done property altogether.

Should I add it to both or leave it out of both?

Reply:
Please add it to both!